src/app/components/query-explainer/query-explainer.component.html

```diff
@@ -0,0 +1,243 @@
+<ng-container *ngIf="verifyResponse">
+    <div
+        class="alert alert-info message-success mx-4 mt-4 fs-18 w-50 mx-auto d-flex align-items-center justify-content-center"
+        role="alert"
+        *ngIf="verifyResponse?.ok"
```

Comment:
You've checked `verifyResponse` in the upper tag. If you are here, it cannot be undefined or null.
Correct everywhere.

Reply:
Fixed